sql/expression_value.cc

```diff
@@ -2608,6 +2608,68 @@ getNestedColumn(const ColumnName & columnName, const VariableFilter & filter) co
     else return *val;
 }
 
+ExpressionValue
+ExpressionValue::
+coerceToEmbedding() const
+{
+    if (type_ == Type::EMBEDDING)
+        return *this;
+    else if (type_ != Type::STRUCTURED)
```

Comment:
In Tensorflow, a scalar is simply an embedding of dimension size zero.  We should consider if this is what we want to do.